### PR TITLE
OSM Asia community

### PR DESCRIPTION
**Pull request by naveenpf**
updated mailing list and telegram group of Asia community

### CODE REVIEW

File: resources/asia/osm-asia-mailing-list.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "OSM-Asia-mailinglist",
+  "featureId": "asia_full",
+  "type": "mailinglist",
+  "name": "OpenStreetMap Asia Mailinglist",
+  "description": "Talk-in is the official Mailinglist for Asian Community",
```
Comment (seav) [inline resources/asia/osm-asia-mailing-list.json]:
```suggestion
   "description": "Talk-asia is the official Mailinglist for Asian Community",
```

File: resources/asia/osm-asia-mailing-list.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "OSM-Asia-mailinglist",
+  "featureId": "asia_full",
```
Comment (bhousel) [inline resources/asia/osm-asia-mailing-list.json]: I think there is no `asia_full` geojson feature to link to.. Should we add one?

Comment (naveenpf) [reply resources/asia/osm-asia-mailing-list.json]: Yes..

Comment (bhousel) [reply resources/asia/osm-asia-mailing-list.json]: Will you be adding it, or do you want me to?

Comment (bhousel) [reply resources/asia/osm-asia-mailing-list.json]: I added it in 7665810

**Pull request merged**